### ISSUE TITLE
External subtitles discovered but failing to load because of an error

### ISSUE THREAD

**Issue by entr** [croneter/PlexKodiConnect#378]
Same Plex server, different clients. Working client - Linux Mint 18 (xenial), KODI 17.6 external subtitles are correctly downloaded and saved with nice names, using the lang codes like `subtitle00.bul.srt`, `subtitle01.bul.srt` etc.

Not working client, LibreELEC 8 (KODI 17.4) - I end up with the the error - `OpenStream: unable to create subtitle parser`.

I looked into logs. This happens on LibreELEC:

```
 tail -f ~/.kodi/temp/kodi.log | grep -s "PLEX\.PlexAPI:"
18:09:16.230 T:2518082464   DEBUG: PLEX.PlexAPI: Writing temp subtitle to /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle00.български език.srt
18:09:16.231 T:2518082464   DEBUG: PLEX.PlexAPI: Need to slugify the filename /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle00.български език.srt
18:09:16.234 T:2518082464  NOTICE: PLEX.PlexAPI: Found external subs: [u'/storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle00.\u0431\u044a\u043b\u0433\u0430\u0440\u0441\u043a\u0438 \u0435\u0437\u0438\u043a.srt']
```

Listing the temp folder:

```
LibreELEC:~/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp # ls -la
total 75
drwxr-xr-x    2 root     root          1024 Dec 27 17:44 .
drwxr-xr-x    3 root     root          1024 Dec 27 14:14 ..
-rw-r--r--    1 root     root         43008 Dec 27 18:09 subtitle00. .srt
-rw-r--r--    1 root     root         31716 Dec 27 17:46 subtitle01. .sub
```

My LibreELEC was localized in Bulgarian, so I tryed to reset all region and locale settings, deleted eveything locale related. It's in plain English now. Did a PKC db reset afterwards to no avail.

Let me know if full logs are needed.

**Comment by croneter** [comment]
Does a PKC subtitle download work with the **working** client for **exactly** the same movie, the same subtitle? Kodi can sometimes not open certain miss-coded subtitle files.

**Comment by entr** [comment]
Yes, same files both movie and srt. I believed it's their naming that causes the issue, not the contents encoding.

**Comment by croneter** [comment]
Please share the entire debug log of the one not working

Am 27.12.2017 6:27 nachm. schrieb "Zlatko Zlatev" <notifications@github.com

**Comment by entr** [comment]
Here https://pastebin.com/pmuzANQN

I think the lines of interest are 891-893

**Comment by Pavuucek** [comment]
@entr Your problem appears to be a bug which was already fixed (#342). You need to update to a later version of PKC manually as described [here](https://github.com/croneter/PlexKodiConnect/wiki/Update-PKC-Repository) and your subtitles should start working again.

**Comment by entr** [comment]
@Pavuucek yes, that was it but I'm already using the latest repo 1.0.2 and I can see only 3 versions available, latest being 1.8.16. More disturbingly the old repo seems to be still working on my linux machine. It's crazy.
